admin/services/data-access/check-window.data.service.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const moment = require('moment')
+const CheckWindow = require('../../models/check-window')
+
+const checkWindowDataService = {
+
+  /**
+   * Fetch check window document by id.
+   * @param id
+   * @returns {Promise.<*>}
+   */
+  fetchCheckWindow: async (id) => {
+    let checkWindow
+    checkWindow = await CheckWindow.findOne({'_id': id, 'deleted': false}).exec()
```

Comment:
We should actually just do

```
return CheckWindow.findOne({'_id': id, 'deleted': false}).exec()
```

Anything calling an async function will will await'ing anyway: https://eslint.org/docs/rules/no-return-await  This also applies to other methods here.

Reply:
@jon-shipley Done, however, do you realise we should modify this syntax all across the site? It's a common pattern we've been using so far.